server/service/src/requisition/request_requisition/insert_program.rs

```diff
@@ -129,6 +132,18 @@ fn validate(
         return Err(OutError::SupplierNotValid);
     }
 
+    if check_exceeded_max_orders_for_period(CheckExceededOrdersForPeriod {
+        connection,
+        program_id: &program_setting.program_requisition_settings.program_row.id,
+        period_id: &input.period_id,
+        program_order_type_id: &input.program_order_type_id,
+        max_orders_per_period: i64::from(order_type.order_type.max_order_per_period),
+        requisition_type: RequisitionType::Response,
+    })? == true
```

Comment:
Doesn't need == true. The function is already returning a bool, so if true return error (: